src/wp-includes/interactivity-api/class-wp-interactivity-api-directives-processor.php

```diff
@@ -180,6 +180,39 @@ private function get_balanced_tag_bookmarks() {
 		return array( $opener_tag, $closer_tag );
 	}
 
+	/**
+	 * Skips processing the content between tags.
+	 *
+	 * It positions the cursor in the closer tag of the foreign element, if it
+	 * exists.
+	 *
+	 * This function is intended to skip processing SVG and MathML inner content
+	 * instead of bailing out the whole processing.
+	 *
+	 * @since 6.5.0
+	 *
+	 * @access private
+	 *
+	 * @return bool Whether the foreign content was successfully skipped.
+	 */
+	public function jump_to_tag_closer(): bool {
```

Comment:
Just realized that `skip` is the terminology used in the tag processor and also in the description so it would probably be better here.

```suggestion
	public function skip_to_tag_closer(): bool {
```

Reply:
That's true.